package.json

```diff
@@ -85,24 +94,25 @@
     "ansi-regex": "^5.0.1"
   },
   "main": "lib/cql",
+  "types": "lib/cql.d.ts",
   "directories": {
     "lib": "lib"
   },
   "scripts": {
     "prepublish": "npm run build --scripts-prepend-node-path && npm run build:browserify --scripts-prepend-node-path",
```

Comment:
I noticed a npm warning when this was getting installed. Looks like this should be `prepare` now.

```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
```